### PR TITLE
[FormWrapper] save button props

### LINKED ISSUE #921: [FormWrapper] Allow button type customization
Current the type is SECONDARY, without prop to customize

### PR DESCRIPTION
**Pull request by MateusStanki**
## Closes Issues

* Closes #921 #922 

### Reviewer 1

* [ ] Functions properly
* [ ] Good readability, clarity and overall quality
* [ ] Has tests
* [ ] Has docs

### Reviewer 2

* [ ] Functions properly
* [ ] Good readability, clarity and overall quality
* [ ] Has tests
* [ ] Has docs

### CODE REVIEW

**Comment by vNakamura** [comment]
Hey, @MateusStanki  can you add a way to remove the save button? I'll need this for the Phone Input and I think it would fit better here than in my PR.

File: packages/form-wrapper/README.md [hunk 1]
```diff
@@ -27,17 +27,18 @@ Output:
...
-| id           | string   | id attribute given to the wrapper and to the save and cancel actionables (with `-save` and `-cancel` sufixes) |          | form-wrapper |
+| propery         | type     | description                                                                                                     | required | default      |
+| --------------- | -------- | --------------------------------------------------------------------------------------------------------------- | -------- | ------------ |
+| title           | string   | The text to be used as the title                                                                                |          |              |
+| cancelLabel     | string   | The text to be used in the Cancel link                                                                          |          | Cancel       |
+| saveLabel       | string   | The text to be used by the Save button                                                                          |          | Save         |
+| saveButtonProps | object   | The props for Save button ([check Button](https://github.com/CraveFood/farmblocks/tree/master/packages/button)) |          | SECONDARY    |
```
Comment (alcferreira) [inline packages/form-wrapper/README.md]: default value is actually undefined

File: packages/form-wrapper/src/FormWrapperFooter.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      fluid
+      {...saveButtonProps}
+    >
+      {saveLabel}
+    </Button>
+  </Footer>
+);
+
+FormWrapperFooter.defaulProps = {
+  cancelLabel: "Cancel",
```
Comment (alcferreira) [inline packages/form-wrapper/src/FormWrapperFooter.js]: either add a default `saveLabel` here or make it required

Comment (alcferreira) [reply packages/form-wrapper/src/FormWrapperFooter.js]: Actually this cancelLabel shouldn't be here, it should be on FormWrapperHeader instead and vice-versa

**Pull request merged**